### PR TITLE
Config update as dict

### LINKED ISSUE #1033: Config endpoint update

### PR DESCRIPTION
**Pull request by na3d**
Closes #1033

### CODE REVIEW

File: src/linodes/linode/settings/layouts/EditConfigPage.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ export function getDiskSlots(fromConfig = false) {
   const diskSlots = [];
```
Comment (mparke) [inline src/linodes/linode/settings/layouts/EditConfigPage.js]: if this does not have empty entries, can be `const diskSlots = Object.values(disks);` ?

Comment (eatonphil) [inline src/linodes/linode/settings/layouts/EditConfigPage.js]: It does have empty entries.

**Comment by eatonphil** [comment]
I requested some other changes to make sure Add / Remove have not been crippled until we address it specifically.

**Comment by na3d** [comment]
please test this

**Comment by na3d** [comment]
retest this please

File: src/linodes/linode/settings/layouts/EditConfigPage.js (hunk 1 above)
Comment (mparke) [inline src/linodes/linode/settings/layouts/EditConfigPage.js]: okay, more appropriate question is then, why does it have empty entries?

Comment (eatonphil) [inline src/linodes/linode/settings/layouts/EditConfigPage.js]: See the example on the right https://developers.linode.com/reference/#object-linode_config

**Pull request merged**